### PR TITLE
better Chinese translation

### PR DESCRIPTION
**Pull request by AndyChenW**
@kadirsekha 

A better Chinese translation.

### CODE REVIEW

File: i18n/po/zh.po [hunk 1]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: www/views/preferences.html:45
+#: www/views/preferences.html:45
 msgid "If enabled, all sensitive information (private key and recovery phrase) and actions (spending and exporting) associated with this wallet will be protected."
-msgstr "如果启用,将保护所有的敏感信息(私人密钥和恢复短语) 和与这个钱包关联的操作 (支出和出口)."
+msgstr "如果启用,将保护所有的敏感信息(私人密钥和恢复短语) 和与这个钱包关联的操作 (支出和出口)"
 
-#: www/views/advancedSettings.html:23
+#: www/views/advancedSettings.html:23
 msgid "If enabled, the Recent Transactions card - a list
```
Comment (hwind) [inline i18n/po/zh.po]: 我们期待听到你的声音？

Comment (hwind) [inline i18n/po/zh.po]: 恢复词数量不正确

File: i18n/po/zh.po [hunk 2]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
-msgstr "5 星评级帮助我们进入更多的人手, {{appName}} 和更多的用户意味着的的资源可以致力于应用!"
+msgstr "5 星评级帮助 {{appName}} 获得和投入更多的资源!"
 
-#: www/views/mercadoLibre.html:18
-#: www/views/mercadoLibre.html:40
+#: www/views/mercadoLibre.html:18
+#: www/views/mercadoLibre.html:40
 msgid "<b>Only</b> redeemable on Mercado Livre (Brazil)"
-msgstr "<b>Only</b>可在悔尔卡多里弗(巴西) 兌換"
+msgstr "<b>仅</b>可在悔尔卡多里弗(巴西) 兌換"
```
Comment (hwind) [inline i18n/po/zh.po]: 兑换

File: i18n/po/zh.po [hunk 3]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
-#: www/views/modals/wallet-balance.html:18
+#: www/views/modals/wallet-balance.html:18
 msgid "All of your bitcoin wallet balance may not be available for immediate spending."
-msgstr "所有您的比特币的钱包余额可能无法用于直接支出。"
+msgstr "所有你的比特币的钱包余额可能无法用于直接支出。"
 
-#: www/views/tab-receive.html:25
+#: www/views/tab-receive.html:25
 msgid "All signing devices must be added to this multisig wallet before bitcoin addresses can be created."
 msgstr "比特币地址可以创建之前，签名的所有设备都必须都添加到这个对数钱包。"
```
Comment (hwind) [inline i18n/po/zh.po]: 对数钱包--》多重签名钱包

File: i18n/po/zh.po [hunk 4]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
-#: www/views/tab-settings.html:79
+#: www/views/preferencesAltCurrency.html:4
+#: www/views/tab-settings.html:79
 msgid "Alternative Currency"
 msgstr "替代货币"
 
-#: src/js/controllers/buyAmazon.js:98
+#: src/js/controllers/buyAmazon.js:98
 msgid "Amazon.com is not available at this moment. Please try back later."
 msgstr "亚马逊服务器暂时无法访问，请稍候再试。"
```
Comment (hwind) [inline i18n/po/zh.po]: Amazon.com should not be translated.

File: i18n/po/zh.po [hunk 5]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: src/js/services/bwcError.js:110
+#: src/js/services/bwcError.js:110
 msgid "Amount below minimum allowed"
 msgstr "数额低于最低允许值"
 
-#: src/js/controllers/confirm.js:216
+#: src/js/controllers/confirm.js:216
 msgid "Amount too big"
 msgstr "量太大"
```
Comment (hwind) [inline i18n/po/zh.po]: 数额太大？

File: i18n/po/zh.po [hunk 6]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: www/views/includes/walletHistory.html:31
+#: www/views/includes/walletHistory.html:31
 msgid "Amount too low to spend"
 msgstr "數額低於可消費額"
```
Comment (hwind) [inline i18n/po/zh.po]: 数额低于可消费额

File: i18n/po/zh.po [hunk 7]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: www/views/addresses.html:94
+#: www/views/addresses.html:94
 msgid "Approximate Bitcoin network fee to transfer wallet's balance (with normal priority)"
 msgstr "近似的比特币网络费用将钱包的余额转移 (与正常优先级)"
 
-#: www/views/backupWarning.html:10
+#: www/views/backupWarning.html:10
 msgid "Are you being watched?"
 msgstr "你正在被监视吗？"
```
Comment (hwind) [inline i18n/po/zh.po]: 有旁观者吗？

File: i18n/po/zh.po [hunk 8]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
+#: www/views/modals/txp-details.html:36
 msgid "Broadcast Payment"
 msgstr "广播支付"
 
-#: src/js/controllers/modals/txpDetails.js:64
-#: src/js/controllers/tx-details.js:81
+#: src/js/controllers/modals/txpDetails.js:64
+#: src/js/controllers/tx-details.js:81
 msgid "Broadcasted"
 msgstr "广播商"
```
Comment (hwind) [inline i18n/po/zh.po]: 已经广播？

File: i18n/po/zh.po [hunk 9]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
-msgstr "點擊以寄出"
-
-#: www/views/customAmount.html:4
-#: www/views/modals/mercadolibre-card-details.html:3
-#: www/views/modals/paypro.html:4
-#: www/views/modals/pin.html:3
-#: www/views/modals/search.html:3
-#: www/views/modals/wallet-balance.html:3
-#: www/views/modals/wallets.html:5
+msgstr "点击以寄出"
```
Comment (hwind) [inline i18n/po/zh.po]: 点击以发送？

File: i18n/po/zh.po [hunk 10]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: www/views/tab-import-phrase.html:2
+#: www/views/tab-import-phrase.html:2
 msgid "Could not access the wallet at the server. Please check:"
 msgstr "无法访问服务器上的钱包。请确认︰"
 
-#: src/js/controllers/buyAmazon.js:102
+#: src/js/controllers/buyAmazon.js:102
 msgid "Could not access to Amazon.com"
 msgstr "不能访问到亚马逊网站"
```
Comment (hwind) [inline i18n/po/zh.po]: Amazon.com should not be translated

File: i18n/po/zh.po [hunk 11]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: src/js/services/profileService.js:511
+#: src/js/services/profileService.js:511
 msgid "Could not access wallet"
 msgstr "无法访问钱包"
 
-#: src/js/controllers/confirm.js:210
+#: src/js/controllers/confirm.js:210
 msgid "Could not add message to imported wallet without shared encrypting key"
 msgstr "无法将消息添加到进口钱包没有共享的加密密钥"
```
Comment (hwind) [inline i18n/po/zh.po]: 没有共享的加密密钥无法将消息添加到导入的钱包

File: i18n/po/zh.po [hunk 12]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: src/js/controllers/modals/txpDetails.js:181
+#: src/js/controllers/modals/txpDetails.js:181
 msgid "Could not delete payment proposal"
 msgstr "无法删除支付提议"
 
-#: src/js/controllers/cashScan.js:117
+#: src/js/controllers/cashScan.js:117
 msgid "Could not duplicate"
 msgstr "無法复制"
```
Comment (hwind) [inline i18n/po/zh.po]: 无法复制

File: i18n/po/zh.po [hunk 13]
```diff
@@ -13,3618 +13,3625 @@ msgstr ""
...
 
-#: src/js/services/feeService.js:73
+#: src/js/services/feeService.js:73
 msgid "Could not get dynamic fee"
 msgstr "不能动态费用"
```
Comment (hwind) [inline i18n/po/zh.po]: 无法获得动态费用

File: i18n/po/zh.po (hunk 1 above)
Comment (hwind) [inline i18n/po/zh.po]: 马卡杜勒比礼品卡服务现在不能使用，请稍后再试

Comment (hwind) [inline i18n/po/zh.po]: 商户的消息

Comment (hwind) [inline i18n/po/zh.po]: 还没有联系人

**Comment by hwind** [comment]
Looks good. Appreciate for your work.

**Comment by vuduy67215** [comment]
Vào 07-02-2018 1:05, "konstantin19900" <notifications@github.com> đã viết:

*@konstantin19900* approved this pull request.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Bitcoin-com/Wallet/pull/32#pullrequestreview-94445663>,
or mute the thread

**Comment by vuduy67215** [comment]
Vào 08-02-2018 0:43, "phim hay" <vuduy67215@gmail.com> đã viết:

**Comment by dannychot** [comment]
Tank toi body



Envoyé depuis mon téléphone intelligent Samsung Galaxy.


-------- Message d'origine --------
De : Jean-Baptiste Dominguez <notifications@github.com>
Date : 18-05-16 02 h 06 (GMT-05:00)
À : Bitcoin-com/Wallet <Wallet@noreply.github.com>
Cc : dannychot <dannychot@hotmail.com>, Comment <comment@noreply.github.com>
Objet : Re: [Bitcoin-com/Wallet] better Chinese translation (#32)


Closed #32<https://github.com/Bitcoin-com/Wallet/pull/32>.

**Comment by dannychot** [comment]
Merci



Envoyé depuis mon téléphone intelligent Samsung Galaxy.


-------- Message d'origine --------
De : FaDee1 <notifications@github.com>
Date : 18-05-10 16 h 05 (GMT-05:00)
À : Bitcoin-com/Wallet <Wallet@noreply.github.com>
Cc : dannychot <dannychot@hotmail.com>, Comment <comment@noreply.github.com>
Objet : Re: [Bitcoin-com/Wallet] better Chinese translation (#32)


@FaDee1 approved this pull request.

**Comment by dannychot** [comment]
Yes yes yes



Envoyé depuis mon téléphone intelligent Samsung Galaxy.


-------- Message d'origine --------
De : jess2929 <notifications@github.com>
Date : 18-04-23 12 h 16 (GMT-05:00)
À : Bitcoin-com/Wallet <Wallet@noreply.github.com>
Cc : dannychot <dannychot@hotmail.com>, Comment <comment@noreply.github.com>
Objet : Re: [Bitcoin-com/Wallet] better Chinese translation (#32)


@jess2929 approved this pull request.

**Pull request closed without merging**